examples/mistral/mistral-dpo.yml

```diff
@@ -0,0 +1,96 @@
+#Note that we are switching from the regular chat template to chatml. 
+#If you experience problems with the special tokens, training for more epochs can help. 
+#After training, merge the model before inference otherwise you might
+#face problems with the special tokens.
```

Comment:
This file may have been adding accidentally

Reply:
Yeah sorry I am still testing some things. This pull request was just to see the pytest. You can ignore this pull request I will make a new one soon